bundles/org.eclipse.ui.workbench/Eclipse UI/org/eclipse/ui/IWorkbenchPreferenceConstants.java

```diff
@@ -705,4 +705,13 @@ public interface IWorkbenchPreferenceConstants {
 	 * @since 3.130
 	 */
 	String LARGE_VIEW_LIMIT = "largeViewLimit"; //$NON-NLS-1$
+
+	/**
+	 * <p>
+	 * Whether the UI updates on runtime during dpi change
+	 * </p>
```

Comment:
It's not only about updating the UI, it's actually about adapting to DPI changes
```suggestion
	 * <p>
	 * Whether the UI adapts to DPI changes at runtime
	 * </p>
```
Please also flag this preference as "experimental" and state that it only affects Windows.